core/frontend/src/tools/AccuDrawViewportUI.ts

```diff
@@ -41,44 +41,71 @@ export class AccuDrawViewportUI extends AccuDraw {
     /** Layout controls in a single row horizontally instead of in columns vertically as an option when using fixed location. */
     horizontalArrangement: false,
     /** When controls follow the cursor, the X and Y offsets applied to the current point to position the top left (values in inches based on screen DPI) */
-    cursorOffset: { x: .4, y: .1 },
+    cursorOffset: { x: 0.4, y: 0.1 },
     /** Replace "^", ";", and ".." with "°" or ":" for easier input. */
     simplifiedInput: true,
     /** Enable simple math operations not supported by quantity parser. */
     mathOperations: true,
-    /** Number of visible characters to show in text input fields. */
-    fieldSize: 12,
-    /** Row spacing of text input fields for vertical arrangement. */
-    rowSpacingFactor: 1.2,
-    /** Column spacing of text input fields and buttons for horizontal arrangement. */
-    columnSpacingFactor: 1.1,
-    /** Corner radius of text input fields and locks buttons. */
-    borderRadius: "0.5em",
-    /** Background color of unfocused text input fields and unlocked buttons. */
-    backgroundColor: "rgba(150, 150, 150, 0.5)",
-
+    field: {
+      /** Number of visible characters to show in text input fields. */
+      size: 12,
+      height: "var(--iui-component-height-small)",
+    },
+    border: {
+      width: "1px",
+      style: "solid",
+      /** Corner radius of text input fields and locks buttons. */
+      radius: "var(--iui-border-radius-1)",
+    },
     /** Settings specific to text input fields and lock button labels. */
     text: {
       /** Font family to use for text input field values and button labels. */
-      fontFamily: "sans-serif",
```

Comment:
There's no fallback for these? Did you try this with display-test-app where these css variables aren't defined?